platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -88,34 +86,14 @@ protected Object doExecute() {
 
                 outputWarningMessage(String.format(FAILED_EXPORT_MESSAGE, exportDirectory));
             }
-        } catch (Exception e) {
```

Comment:
Do you think we should do anything for Runtime exceptions?  For example, isn't is possible for the ConfigurationMigrationService.export() to throw a MigrationException?